website/cue/reference/components/sources/eventstoredb_metrics.cue

```diff
@@ -143,10 +143,13 @@ components: sources: eventstoredb_metrics: {
 		}
 	}
 	telemetry: metrics: {
-		events_in_total:                 components.sources.internal_metrics.output.metrics.events_in_total
-		http_request_errors_total:       components.sources.internal_metrics.output.metrics.http_request_errors_total
-		parse_errors_total:              components.sources.internal_metrics.output.metrics.parse_errors_total
-		processed_bytes_total:           components.sources.internal_metrics.output.metrics.processed_bytes_total
-		component_received_events_total: components.sources.internal_metrics.output.metrics.component_received_events_total
+		events_in_total:                      components.sources.internal_metrics.output.metrics.events_in_total
+		http_request_errors_total:            components.sources.internal_metrics.output.metrics.http_request_errors_total
```

Comment:
Noting for future work: It would be good to mark deprecated metrics like this as such once they are all converted to the component spec.

Reply:
@jszwedko raised the deprecation subject on another PR. I will deal with this point.